src/app/models/genes.ts

```diff
@@ -29,17 +28,16 @@ export interface NominatedTarget {
 
 export interface MedianExpression {
   ensembl_gene_id: string;
```

Comment:
Should `ensemble_gene_id` also be removed from this interface?

Reply:
Good catch!  I only removed it on the api side.